splash/render_options.py

```diff
@@ -248,11 +251,23 @@ def get_common_params(self, js_profiles_path):
             # 'lua': self.get_lua(),
         }
 
+    def get_image_params(self):
+        return {
+            'width': self.get_width(),
+            'height': self.get_height(),
+            'scale_method': self.get_scale_method()
+        }
+
     def get_png_params(self):
-        return {'width': self.get_width(), 'height': self.get_height(),
-                'scale_method': self.get_scale_method()}
+        return self.get_image_params()
+
+    def get_jpeg_params(self):
+        params = {'quality': self.get_quality()}
```

Comment:
This seems a bit tricky-ish. Why do we want quality for one of the image rendering methods and disregard it for the other one?